### PR TITLE
Add nebari 101

### PR DESCRIPTION
**Pull request by kcpevey**
## Reference Issues or PRs

Resolves https://github.com/nebari-dev/nebari-docs/issues/18

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense, and no information is missing.

### Any other comments?

I had originally thought I would take this further, but once I had this written out, I saw that working with the individual integrations should be their own pages. Not sure how others feel about this approach or if there are things I'm missing.

### CODE REVIEW

**Comment by pavithraes** [comment]
@kcpevey The first section in the 101 is similar to [How to authenticate using Keycloak and launch your server](https://www.nebari.dev/docs/how-tos/login-keycloak/), do we need these in two places?

I had understood this page as the one-page complete reference for Nebari, like what https://spacy.io/usage/spacy-101 is intended to be. I might be wrong here, so I want to confirm the goal for the page. I see you introduce:
>  This guide is intended to walk new users through the process of getting started using Nebari as a development platform. 

In this case, maybe some sections from https://deploy-preview-216--nebari-docs.netlify.app/docs/welcome can be moved to the 101 page?

**Comment by pavithraes** [comment]
@kcpevey Thanks for opening this PR!

The first section in the 101 is similar to [How to authenticate using Keycloak and launch your server](https://www.nebari.dev/docs/how-tos/login-keycloak/), do we want to remove the `login-keycloak` page in favor of this section, or vice-versa, or keep both?

I had understood this page as the one-page complete reference for Nebari, like what https://spacy.io/usage/spacy-101 is intended to be. I might be wrong here, so I want to confirm the goal for the page. :) I see you introduce:
>  This guide is intended to walk new users through the process of getting started using Nebari as a development platform. 

In this case, maybe some sections from https://deploy-preview-216--nebari-docs.netlify.app/docs/welcome can be moved to the 101 page?

File: docs/docs/get-started/nebari-101.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+description: An introduction to using Nebari as a dev platform
+---
+
+import nebari_logo from '/logo/Nebari-logo-square.png';
+
+# Nebari 101 <img src={nebari_logo} style={{height: 60, border: 0, boxShadow: "none", verticalAlign: "text-top"}} />
+
+This guide is intended to walk new users through the process of getting started
+using Nebari as a development platform. You may be here as a software developer,
+or
```
Comment (pavithraes) [inline docs/docs/get-started/nebari-101.mdx:12]:
```suggestion
data scientist, or an administrator.
```

**Comment by trallard** [comment]
@kcpevey  what is the status of this PR?

**Comment by kcpevey** [comment]
@trallard I'm not sure what this page needs to be. My initial idea has been superseded by other documents so I don't know exactly what to included here.

**Comment by trallard** [comment]
Ok it does seem like this was introduced in other docs. Will close this for now and reassess as we complete docs work

**Pull request closed without merging**